CaptainMumo/stackoverflow-lite
# #162571650 Allow user to accept or decline answer

#### What does this PR do?
Add a feature that allows a user to accept or decline an answer as being satisfactory
#### Description of Task to be completed?
- On the fieldset for displaying an answer, add two buttons namely 'Accept' and 'Decline'
- Style the buttons to enable easy recognition of the answer's state
#### How should this be manually tested?
1. Clone this repo, [git@github.com:CaptainMumo/stackoverflow-lite.git](url)
2. Git checkout ft-user-accept-answer-162571650
3. Open UI folder in a text/code editor
4. Run index.html on a browser, click on a question and view the buttons below the answers and/or
5. Run question.html on a browser and see the state buttons
#### Any background context you want to provide?
Users will only be able to accept or decline an answer if they are the author of the answer
#### What are the relevant pivotal tracker stories?
[https://www.pivotaltracker.com/story/show/162571650](url)
#### Screenshots
![image](https://user-images.githubusercontent.com/39964147/50058674-d27d7f80-018c-11e9-99fd-07681b031885.png)
